lms/djangoapps/courseware/management/commands/tests/test_dump_course.py

```diff
@@ -205,15 +204,6 @@ def check_export_file(self, tar_file):  # pylint: disable=missing-docstring
         assert_in('edX-simple-2012_Fall/sequential/Lecture_2.xml', names)
 
 
-class CommandsXMLTestCase(CommandsTestBase):
```

Comment:
I don't know enough about the modulestore to really evaluate this, but I think so?

@doctoryes - is testing with a mixed module store that includes XML courses relevant anymore? Is that what this test is doing?